### ISSUE TITLE
Import issue - TS2307: Cannot find module 'chromiumly' or its corresponding type declarations.

### ISSUE THREAD

**Issue by solshark** [cherfia/chromiumly#279]
Does it have TS support for real?

```bash
pnpm i chromiumly
```

```typescript
import { HtmlConverter } from 'chromiumly';
```

TS2307: Cannot find module 'chromiumly' or its corresponding type declarations.

**Comment by cherfia** [comment]
@solshark Thank you so much for your report. The bug has been addressed and resolved in version [2.3.1](https://github.com/cherfia/chromiumly/releases/tag/v2.3.1). Could you kindly update and let me know if it resolves your problem? 
Cheers!

**Comment by cherfia** [comment]
@solshark I recently tweaked how to publish the library, using the [release-it ](https://github.com/release-it/release-it) for a smoother release and publishing to GitHub and NPM. However, I unintentionally forgot to throw in the build step in the [release-it config](https://github.com/cherfia/chromiumly/commit/16e852a4392bdf46fbb876e43ca6e1a4a7bd061d), and that's what caused the bug you encountered.
